tools/nightly-e2e-tests/monitors-gen/terraform.tfstate

```diff
@@ -0,0 +1,225 @@
+{
+  "version": 4,
+  "terraform_version": "0.15.4",
+  "serial": 18,
+  "lineage": "763b9aae-5403-efe4-102c-e6d6877ad82d",
+  "outputs": {},
+  "resources": [
+    {
+      "mode": "managed",
+      "type": "datadog_monitor",
+      "name": "logs_logger_debug_log_data",
+      "provider": "provider[\"registry.terraform.io/datadog/datadog\"]",
+      "instances": [
+        {
+          "schema_version": 0,
+          "attributes": {
+            "enable_logs_sample": false,
+            "escalation_message": "\u003cnil\u003e",
+            "evaluation_delay": 0,
+            "force_delete": null,
+            "groupby_simple_monitor": false,
+            "id": "38372071",
+            "include_tags": true,
+            "locked": false,
+            "message": "@maciek.grzybowski@datadoghq.com\n@mert.buran@datadoghq.com",
+            "monitor_threshold_windows": [],
+            "monitor_thresholds": [
+              {
+                "critical": "1",
+                "critical_recovery": "",
+                "ok": "",
+                "unknown": "",
+                "warning": "",
+                "warning_recovery": ""
+              }
+            ],
+            "name": "[RUM] [iOS] Nightly - logs_logger_debug_log: number of logs is below expected value",
+            "new_host_delay": 300,
+            "no_data_timeframe": 0,
+            "notify_audit": false,
+            "notify_no_data": false,
+            "priority": 0,
+            "query": "logs(\"service:com.datadog.ios.nightly @test_method_name:logs_logger_debug_log status:debug\").index(\"*\").rollup(\"count\").last(\"1d\") \u003c 1",
+            "renotify_interval": 0,
+            "require_full_window": true,
+            "restricted_roles": [],
+            "tags": [
+              "env:instrumentation",
+              "service:com.datadog.ios.nightly",
+              "source:ios",
+              "team:rumm"
+            ],
+            "timeout_h": 0,
+            "type": "log alert",
+            "validate": null
+          },
+          "sensitive_attributes": [],
+          "private": "bnVsbA=="
```

Comment:
what `private` means in the `tfstate` schema? can it leak sensitive info somehow? We need to be careful with this file by looking at this page https://www.terraform.io/docs/language/state/sensitive-data.html

Reply:
n/a: https://github.com/DataDog/dd-sdk-ios/pull/520#discussion_r653454702